app/screens/user_profile/title/avatar.tsx

```diff
@@ -18,19 +18,26 @@ const AnimatedFastImage = Animated.createAnimatedComponent(FastImage);
 type Props = {
     enablePostIconOverride: boolean;
     forwardRef?: React.RefObject<any>;
+    imageSize?: number;
     user: UserModel;
     userIconOverride?: string;
 }
 
-const styles = StyleSheet.create({
-    avatar: {
-        borderRadius: 48,
-        height: 96,
-        width: 96,
-    },
-});
+const DEFAULT_IMAGE_SIZE = 96;
+
+const getStyles = (size?: number) => {
+    return StyleSheet.create({
+        avatar: {
+            borderRadius: 48,
+            height: size || DEFAULT_IMAGE_SIZE,
+            width: size || DEFAULT_IMAGE_SIZE,
+        },
+    });
+};
+
+const UserProfileAvatar = ({enablePostIconOverride, forwardRef, imageSize, user, userIconOverride}: Props) => {
+    const styles = useMemo(() => getStyles(imageSize), [imageSize]);
```

Comment:
For my own knowledge, what/why is this optimization?

Reply:
UX wants a different avatar size in the user profile for manage members user profile.